services/user.js

```diff
@@ -326,6 +328,13 @@ class User {
                 newProfile[key] = updatedUser[key];
             });
 
+            const aUser = user[0];
```

Comment:
Maybe I haven't fully understood the logic, will it send inactive email whenever the edit user happens?